src/test/java/org/fogbowcloud/manager/requests/api/local/http/VolumeOrdersControllerTest.java

```diff
@@ -55,137 +50,184 @@
 
     @Before
     public void setUp() {
-        this.facade = spy(ApplicationFacade.class);
+        this.facade = Mockito.spy(ApplicationFacade.class);
     }
 
+    // test case: When calling the createVolume() method, it must return a ID of the Order and to
+    // confirm the HttpStatus as Created.
     @Test
     public void createdVolumeTest() throws Exception {
+
+        // set up
         VolumeOrder order = createVolumeOrder();
 
         PowerMockito.mockStatic(ApplicationFacade.class);
-        given(ApplicationFacade.getInstance()).willReturn(this.facade);
-        doReturn(order.getId()).when(this.facade).createVolume(any(VolumeOrder.class), anyString());
+        BDDMockito.given(ApplicationFacade.getInstance()).willReturn(this.facade);
+
+        // exercise
+        Mockito.doReturn(order.getId()).when(this.facade)
+                .createVolume(Mockito.any(VolumeOrder.class), Mockito.anyString());
 
         HttpHeaders headers = getHttpHeaders();
 
         MvcResult result = this.mockMvc.perform(MockMvcRequestBuilders.post(VOLUME_END_POINT)
-                .headers(headers)
-                .accept(MediaType.APPLICATION_JSON)
-                .content(CORRECT_BODY)
+                .headers(headers).accept(MediaType.APPLICATION_JSON).content(CORRECT_BODY)
                 .contentType(MediaType.APPLICATION_JSON)).andReturn();
 
         int expectedStatus = HttpStatus.CREATED.value();
 
         String resultVolumeId = result.getResponse().getContentAsString();
 
-        assertEquals(order.getId(), resultVolumeId);
-        assertEquals(expectedStatus, result.getResponse().getStatus());
+        // verify
+        Mockito.verify(this.facade).createVolume(Mockito.any(VolumeOrder.class),
```

Comment:
Wouldn't it be better to specify the times parameter on verify as well?

Reply:
Ok, fixed!